docs/DOCKER.md

```diff
@@ -65,3 +65,30 @@ $ docker logs lnd-testnet
 
 This is a simple example, it is possible to use any command-line options necessary
 to expose RPC ports, use `btcd` or `bitcoind`, or add additional chains.
+
+## LND Development and Testing
+
+To test the Docker production image locally, run the following from
+the project root:
+
+```
+$ docker build . -t lnd:1.0
+```
+
+To choose a specific branch or tag instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
+
+```
+$ docker build . --build-arg checkout=master -t lnd:1.0
+```
+
+To build the image using the most current tag:
+
+```
+$ docker build . --build-arg checkout=$(git describe --tags `git rev-list --tags --max-count=1`) -t lnd:1.0
```

Comment:
Same as above, should use a different example name than `lnd:1.0`. Maybe `lnd:latest-tag` or something.

Reply:
Cool.  happy to make that update.